src/javascript/expression.rs

```diff
@@ -113,10 +115,66 @@ impl Generator {
     }
 }
 
+fn int(value: &str) -> Document<'_> {
+    value.to_doc()
+}
+
+fn float(value: &str) -> Document<'_> {
+    value.to_doc()
+}
+
 fn string(value: &str) -> Document<'_> {
     value.to_doc().surround("\"", "\"")
 }
 
+fn bin_op<'a>(name: &'a BinOp, left: &'a TypedExpr, right: &'a TypedExpr) -> Output<'a> {
+    // let div_zero = match name {
+    //     BinOp::DivInt | BinOp::ModuloInt => Some("0"),
+    //     BinOp::DivFloat => Some("0.0"),
+    //     _ => None,
+    // };
+    match name {
+        BinOp::And => print_bin_op(left, right, "&&"),
+        BinOp::Or => print_bin_op(left, right, "||"),
+        BinOp::LtInt | BinOp::LtFloat => print_bin_op(left, right, "<"),
+        BinOp::LtEqInt | BinOp::LtEqFloat => print_bin_op(left, right, "<="),
+        // https://dmitripavlutin.com/how-to-compare-objects-in-javascript/
+        // BinOp::Eq => "=:=",
+        // BinOp::NotEq => "/=",
+        BinOp::GtInt | BinOp::GtFloat => print_bin_op(left, right, ">"),
+        BinOp::GtEqInt | BinOp::GtEqFloat => print_bin_op(left, right, ">="),
+        BinOp::AddInt | BinOp::AddFloat => print_bin_op(left, right, "+"),
+        BinOp::SubInt | BinOp::SubFloat => print_bin_op(left, right, "-"),
+        BinOp::MultInt | BinOp::MultFloat => print_bin_op(left, right, "*"),
+        BinOp::DivInt => Ok(Document::String("Math.floor".to_string())
```

Comment:
This doesn't need to be a string, we can use a `str` and avoid the extra heap allocation

Reply:
Ah yes, I think there is to_doc() function that works